### PR TITLE
[DEV APPROVED] Move CollapsableMobile from frontend to Dough

### PR DESCRIPTION
**Pull request by nmiddleweek**
This is new component is used when you need the Collapsable menu only in Mobile devices.

### CODE REVIEW

**Comment by slaywell** [comment]
This would be a good one to tidy up the commit history on. Could you add a short PR description as well?
Only comment on the actual file changes is that there is some extra newlines in the SCSS that we don't need. I'll add a line marker.

File: assets/stylesheets/components/common/_collapsable-mobile.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    }
+  }
+
+  button {
+    text-align: left;
+
+    @include respond-to($mq-m) {
+      display: none;
+    }
+
```
Comment (slaywell) [inline assets/stylesheets/components/common/_collapsable-mobile.scss]: Remove these empty lines?

**Comment by slaywell** [comment]
:+1:

File: assets/stylesheets/lib/_mediaqueries.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
+//Media Query Variables
```
Comment (munckymagik) [inline assets/stylesheets/lib/_mediaqueries.scss]: This looks like a pretty core core file that we must have already had somewhere. Where did this get moved from?

Comment (munckymagik) [inline assets/stylesheets/lib/_mediaqueries.scss]: And is there another PR we should look at where this gets removed from another codebase? Yeast maybe?

Comment (slaywell) [inline assets/stylesheets/lib/_mediaqueries.scss]: @munckymagik - from Yeast. We moved it from Dough to Yeast a while back, but we need it back in Dough now that Dough has media-query dependent stuff

**Comment by munckymagik** [comment]
:+1:

**Comment by nmiddleweek** [comment]
@munckymagik This is the Yeast PR: https://github.com/moneyadviceservice/yeast/pull/12

**Comment by munckymagik** [comment]
All thanks Nick.

**Pull request merged**